inc/commoninjectionlib.class.php

```diff
@@ -783,8 +766,7 @@ private static function findUser($value, $entity)
               WHERE LOWER(`name`) = '" . strtolower($value) . "'
                  OR (CONCAT(LOWER(`realname`),' ',LOWER(`firstname`)) = '" . strtolower($value) . "'
                     OR CONCAT(LOWER(`firstname`),' ',LOWER(`realname`)) = '" . strtolower($value) . "')";
-        /** @phpstan-ignore-next-line */
-        $result = $DB->query($sql); // phpcs:ignore
+        $result = $DB->doQuery($sql);
```

Comment:
This should have been replaced with a properly built query, no a `doQuery` call.

I won't comment on all possible similar issues here; all `doQuery` coll on this plugin must be reviewed.